CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 ## [Unreleased]
 ### Added
 - Technical - Add optional chaining plugin to babel.
+- Tests - Add MySQL min (5.6) and max (8.0) versions.
+- Database Support - Add support for MySQL 8.
 
 ### Changed
 - Technical - Upgrade to babel 7 stable.
+- Technical - Update `mysql2` dependency to 2 (latest MAJOR version).
```

Comment:
Is this a necessary change to support MySQL 8?

Reply:
Yes (and no). The default password system for MySQL 8 is not supported by `mysql2` < 2:

> 2.0.0-alpha1
> MAJOR: new authPlugins api replacing authSwitchHandler, added caching_sha2_password and mysql_native_password as default plugins. Added tests for mysql 8 and ssl. Mysql 8 server now supported with default settings. #1021, #906, #991

Without this, users should do a kind of a hack to downgrade their password system: https://stackoverflow.com/a/50131831/978690

In other words: if we do not upgrade `mysql2`, **users will have to downgrade** their password policy (it is not future-proof).

Reply:
Thanks!
So it is part of the MySQL 8 support if I read you well (if it is not, it means we already support MySQL 8, because no other changes have been made).

As a consequence I would not mention the technical dependency upgrade in the changelog.
The intent is already explained in the "Database Support - Add support for MySQL 8." change description.